src/style.css

```
@@ -0,0 +1,114 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  font-weight: 300;
+  src: url("\fonts\Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  font-weight: 900;
+  src: url("\fonts\Avenir-Heavy.ttf");
+}
+
+.Search {
+  margin-top: 20px;
+}
+
+.SearchBar {
```

```suggestion
.search-bar {
```
it's better to name classes with kebab-case

---

I tried to use the BEM naming convention ([OOCSS and BEM](https://mate-academy.github.io/style-guides/htmlcss.html#oocss-and-bem)), so I choose  PascalCased naming for blocks

---

it looks unusual. expected to see kebab-case in class names, but it depends on the project and company

---

@DarkMistyRoom, should I change to kebab-case in this pull request?